jsapp/js/components/modalForms/formMedia.es6

```diff
@@ -37,20 +31,40 @@ class FormMedia extends React.Component {
    */
 
   componentDidMount() {
-    this.refreshFormMedia();
+    actions.media.loadMedia.completed.listen(this.onGetMediaCompleted);
+    actions.media.uploadMedia.completed.listen(this.onUploadCompleted);
+    actions.media.uploadMedia.failed.listen(this.onUploadFailed);
+    actions.media.deleteMedia.completed.listen(this.onGetMediaCompleted);
+    actions.media.deleteMedia.failed.listen(this.onDeleteMediaFailed);
+
+    actions.media.loadMedia(this.props.asset.uid);
     this.setState({isVirgin: false});
```

Comment:
Shouldn't this `isVirgin: false` be set only after the first `actions.media.loadMedia` is completed? You could set it in the `onGetMediaCompleted`